packages/debugger/src/hash.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
This file is vendored as part of the implementation as part of the `debugger` package to avoid increasing the API surface for now.